### PR TITLE
add style guide blockquote in v-if with v-for

### PR DESCRIPTION
**Pull request by CeceWall**
in the vue style guide, it's an essential rule to avoid use v-if and v-for together, i thought beginners should know this.

### CODE REVIEW

File: src/v2/guide/conditional.md [hunk 1]
```diff
@@ -196,4 +196,6 @@ Generally speaking, `v-if` has higher toggle costs while `v-show` has higher ini
 
 ## `v-if` with `v-for`
 
+> Use v-if and v-for together is **not recommended**. See the style guide for futher infomation [Avoid v-if with v-for](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential)
```
Comment (phanan) [inline src/v2/guide/conditional.md]: I'd suggest something like this instead:

```md
> Use v-if and v-for together is **not recommended**. See the [style guide](/v2/style-guide/#Avoid-v-if-with-v-for-essential) for further information.
```

Notice the wording changes and the two spelling mistake corrections.

Comment (CeceWall) [reply src/v2/guide/conditional.md]: thanks for your change and correcting, your version is much better.

**Comment by CeceWall** [comment]
sorry for didn't get your point, and I push a new commit use p.tip instead.

**Comment by CeceWall** [comment]
does anybody here?

File: src/v2/guide/conditional.md [hunk 2]
```diff
@@ -196,4 +196,6 @@ Generally speaking, `v-if` has higher toggle costs while `v-show` has higher ini
 
 ## `v-if` with `v-for`
 
+<p class="tip">Use v-if and v-for together is **not recommended**. See the [style guide](/v2/style-guide/#Avoid-v-if-with-v-for-essential) for further information.</p>
```
Comment (phanan) [inline src/v2/guide/conditional.md]: Please enclose `v-if` and `v-for` in backticks.

**Comment by phanan** [comment]
@CeceWall Thanks for the ping. One extra little change and I think we're good to go :)

**Comment by CeceWall** [comment]
thanks, and i have finished that

File: src/v2/guide/conditional.md [hunk 3]
```diff
@@ -196,4 +196,6 @@ Generally speaking, `v-if` has higher toggle costs while `v-show` has higher ini
 
 ## `v-if` with `v-for`
 
+<p class="tip">Use `v-if` and `v-for` together is **not recommended**. See the [style guide](/v2/style-guide/#Avoid-v-if-with-v-for-essential) for further information.</p>
```
Comment (phanan) [inline src/v2/guide/conditional.md]: Sorry, but I miss this earlier 😅: It's "Using," not "Use."

**Comment by phanan** [comment]
Thanks!

**Pull request merged**